python/triton/runtime/interpreter.py

```diff
@@ -539,7 +601,8 @@ def _sum(input, axis=None, keepdims=False):
             "_sum_combine": _sum
```

Comment:
In the code above (I can't seem to comment on context lines?) instead of switching on the function's name, how about switching on the function's *object identity*?

```
if combine_fn == tl.standard._elementwise_min
```

This way it will work even if the user defines a function called `_elementwise_min`.

Reply:
Same inside _new_scan.

Reply:
I'll make a change